dot/core/test_helpers.go

```diff
@@ -23,15 +23,20 @@ import (
 	"github.com/ChainSafe/gossamer/dot/network"
 	"github.com/ChainSafe/gossamer/dot/state"
 	"github.com/ChainSafe/gossamer/dot/types"
+	"github.com/ChainSafe/gossamer/lib/common"
 	"github.com/ChainSafe/gossamer/lib/genesis"
 	"github.com/ChainSafe/gossamer/lib/keystore"
 	"github.com/ChainSafe/gossamer/lib/runtime"
 	"github.com/ChainSafe/gossamer/lib/trie"
+
 	"github.com/stretchr/testify/require"
 )
 
-// testGenesisHeader is a test block header
-var testGenesisHeader = &types.Header{
+// TestAuthorityDataKey is the default authority hash used for dot tests
+var TestAuthorityDataKey, _ = common.HexToBytes("0xe3b47b6c84c0493481f97c5197d2554f")
```

Comment:
this isn't a hash, it's the location in the trie of where authority data is fetched from via the runtime

change to "is the location of authority data in the storage trie"

Reply:
kk, will do